### PR TITLE
[KeyVault] Do Not Merge: Review to publish Key Vault preview packages from KvDevPsJson

### PR DESCRIPTION
**Pull request by schaabs**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Core/Microsoft.Azure.KeyVault.Core.csproj [hunk 1]
```diff
@@ -5,8 +5,8 @@
     <Description>Microsoft Azure Key Vault Core Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Core</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.Core</AssemblyName>
-    <VersionPrefix>2.0.4</VersionPrefix>
-    <PackageTags>Microsoft Azure key vault core</PackageTags>
+    <VersionPrefix>2.0.5-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure Key Vault Core</PackageTags>
```
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Core/Microsoft.Azure.KeyVault.Core.csproj]: @schaabs can we add packageRelease Notes as below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Microsoft.Azure.KeyVault.Cryptography.csproj [hunk 2]
```diff
@@ -4,9 +4,9 @@
     <PackageId>Microsoft.Azure.KeyVault.Cryptography</PackageId>
     <Description>Microsoft Azure Key Vault Cryptography Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Cryptography</AssemblyTitle>
-    <VersionPrefix>2.0.5</VersionPrefix>
+    <VersionPrefix>2.0.6-preview</VersionPrefix>
     <AssemblyName>Microsoft.Azure.KeyVault.Cryptography</AssemblyName>
-    <PackageTags>Microsoft Azure key vault cryptography</PackageTags>    
+    <PackageTags>Microsoft Azure Key Vault Cryptography</PackageTags>
```
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Microsoft.Azure.KeyVault.Cryptography.csproj]: @schaabs packageRelseNotes as above

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Extensions/Microsoft.Azure.KeyVault.Extensions.csproj [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <Description>Microsoft Azure Key Vault Extensions Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Extensions</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.Extensions</AssemblyName>
-    <VersionPrefix>2.0.5</VersionPrefix>
+    <VersionPrefix>2.4.0-preview</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Extensions/Microsoft.Azure.KeyVault.Extensions.csproj]: @schaabs packageReleaseNotes missing

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey/Microsoft.Azure.KeyVault.WebKey.csproj [hunk 4]
```diff
@@ -5,25 +5,26 @@
     <Description>Microsoft Azure Key Vault WebKey Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault WebKey</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.WebKey</AssemblyName>
-    <VersionPrefix>2.0.7</VersionPrefix>
-    <PackageTags>Microsoft Azure key vault WebKey</PackageTags>    
+    <VersionPrefix>2.1.0-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure Key Vault WebKey</PackageTags>
```
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey/Microsoft.Azure.KeyVault.WebKey.csproj]: @schaabs package release notes

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Microsoft.Azure.KeyVault.csproj [hunk 5]
```diff
@@ -3,10 +3,10 @@
...
-    <AssemblyTitle>Microsoft Azure key vault</AssemblyTitle>
-    <VersionPrefix>2.3.2</VersionPrefix>
+    <AssemblyTitle>Microsoft Azure Key Vault</AssemblyTitle>
+    <VersionPrefix>2.4.0-preview</VersionPrefix>
     <AssemblyName>Microsoft.Azure.KeyVault</AssemblyName>
-    <PackageTags>Microsoft Azure key vault;Key Vault;REST HTTP client;azureofficial;windowsazureofficial</PackageTags>
+    <PackageTags>Microsoft Azure Key Vault;Key Vault;REST HTTP client;azureofficial;windowsazureofficial</PackageTags>
     <PackageReleaseNotes>This is a preview release of the Azure Key Vault .NET SDK, based on version 2016-10-01 of the Azure Key Vault REST API. For migration guide from version 1.0, please visit https://azure.microsoft.com/en-us/documentation/articles/key-vault-dotnet2api-release-notes/.</PackageReleaseNotes>
```
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Microsoft.Azure.KeyVault.csproj]: @schaabs no updates added to the release notes.
Also using CData will help a little to format your notes. Very soon we will have support for markdown release notes.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Looks good

**Comment by shahabhijeet** [comment]
@schaabs please reopen this PR once you are ready to push changes to this PR. I can help you reopen this PR. Closing it for now.

**Comment by schaabs** [comment]
@shahabhijeet  I've update the package release notes for all packages PTAL

**Comment by shahabhijeet** [comment]
@schaabs please reopen this PR when it's ready to be merged.

**Pull request closed without merging**